src/notifications/components/Notification.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import localStyles from './Notification.css'
```

Comment:
Another not-so-important thing, but I've noticed it a lot: sometimes you name your component styles `localStyles` and sometimes just `styles`. I would encourage you to try to stick to just one of them in the future (probably `styles`)